packages/esm-patient-common-lib/src/lab-resource/types/encounter.ts

```diff
@@ -0,0 +1,65 @@
+import { type OpenmrsResource, type Visit } from '@openmrs/esm-framework';
+import { type Order } from '@openmrs/esm-patient-common-lib';
+
+export interface Encounter {
```

Comment:
I'm inclined to say we should add these instead to [src/types/index.ts](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-common-lib/src/types/index.ts)